import/marc_local.properties

```diff
@@ -72,7 +72,7 @@ author_synonyms = 101a
 publisher = custom(de.unituebingen.ub.ubtools.solrmarcMixin.IxTheoPublisher), getPublishers
 publisher_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.IxTheoPublisher), getPublishersOrUnassigned
 
-topic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600x:610x:611x:630x:648x:650a:650x:651x:655x)
+topic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600x:610x:611x:630x:648x:650a:650x:651x:655x:689abctnp)
```

Comment:
Zusammenanzeigen der Unterfelder t,p,n möglich?
Z.B. |t Bibel |p Genesis |n 1-3
Nur "Bibel" in der Facette macht wenig sinn.